ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -676,13 +676,14 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-registry-s3-cloudfront
-- as: aws-ipi-shared-vpc-phz-f2
+- as: aws-ipi-shared-vpc-freeipa-phz-f14
   cron: 12 4 1,3,5,7,9,11,13,15,17,19,21,23,25,27,29 * *
```

Comment:
Current cron field still uses **f2**, i.e. **f**requency **2** days. You can change by:
```
$ cd ci-operator/config/openshift/openshift-tests-private/tools/
$ ./generate-cron-entry.sh aws-ipi-shared-vpc-freeipa-phz-f14 ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml
cron: 19 16 10,24 * *
```
The new cron value will be printed for use.

Reply:
I tried changing with above script but it's rollback by `make update` 
```
-  cron: 8 14 2,18 * *
+  cron: 12 4 1,3,5,7,9,11,13,15,17,19,21,23,25,27,29 * *
```

Reply:
I just now try, it works for me, no rollback:
```
[xxia@2024-04-09 14:14:06 CST release]$ git checkout -b test-cron-change

[xxia@2024-04-09 14:16:23 CST release]$ ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh aws-ipi-shared-vpc-freeipa-phz-f14 ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml
cron: 19 16 10,24 * *

[xxia@2024-04-09 14:20:06 CST release]$ git diff
...
     workflow: cucushift-installer-rehearse-aws-ipi-registry-s3-cloudfront
-- as: aws-ipi-shared-vpc-phz-f2
-  cron: 12 4 1,3,5,7,9,11,13,15,17,19,21,23,25,27,29 * *
+- as: aws-ipi-shared-vpc-freeipa-phz-f14
+  cron: 19 16 10,24 * *
...

[xxia@2024-04-09 14:33:20 CST release]$ make update

[xxia@2024-04-09 14:42:45 CST release]$ git status
...
        modified:   ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml
        modified:   ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16-periodics.yaml

[xxia@2024-04-09 14:43:04 CST release]$ git diff ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml
...
     workflow: cucushift-installer-rehearse-aws-ipi-registry-s3-cloudfront
-- as: aws-ipi-shared-vpc-phz-f2
-  cron: 12 4 1,3,5,7,9,11,13,15,17,19,21,23,25,27,29 * *
+- as: aws-ipi-shared-vpc-freeipa-phz-f14
+  cron: 19 16 10,24 * *
```

Reply:
Thanks, Now it work for me as well, look like i missed space or did other mistake